ctk/common/src/main/kotlin/org/codice/compliance/verification/core/responses/CoreAuthnRequestProtocolVerifier.kt

```diff
@@ -36,14 +37,13 @@ import org.codice.compliance.verification.core.ResponseVerifier
 import org.codice.compliance.verification.core.SubjectComparisonVerifier
 import org.codice.security.saml.SamlProtocol.Binding.HTTP_POST
 import org.opensaml.saml.saml2.core.AuthnRequest
-import org.w3c.dom.Node
 
 class CoreAuthnRequestProtocolVerifier(private val authnRequest: AuthnRequest,
-                                       samlResponseDom: Node) :
+                                       samlResponseDom: NodeDecorator) :
```

Comment:
✏️ Nit: Could you rename this argument to `samlResponse` like you did in `ResponseVerifier`? It's initially confusing when reading just this class 😅